pureport_client/helpers.py

```diff
@@ -7,12 +7,18 @@
 
 import time
 
+from json import dumps as json_dumps
+from yaml import dump as yaml_dumps
```

Comment:
should this be `dumps` instead of `dump`?

Reply:
these imports and the code that relies on them were just moved from another file for the sake of organization, the yaml library only has a `dump` function and not `dumps`